### PR TITLE
Smoketest adjustment to TLS

### PR DESCRIPTION
**Pull request by JasonTheMain**
## Changes proposed in this pull request:

- Addition of smoketest job changes
- add in missing consume
-

## Security considerations

none

### CODE REVIEW

File: opensearch-jobs.yml [hunk 1]
```diff
@@ -196,7 +200,11 @@ instance_groups:
         - -Dlog4j2.formatMsgNoLookups=true
         migrate_data_path: true
     release: opensearch
-  - name: archiver_syslog
+  - consumes:
```
Comment (markdboyd) [inline opensearch-jobs.yml:203]: How is this related to the changes?

Comment (JasonTheMain) [reply opensearch-jobs.yml:203]: it got deleted when removed by mistake and is needed

**Review by markdboyd — APPROVED** [review]
